graphs/adapter-api/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/api/UndirectedGraphActionFactory.kt

```diff
@@ -5,7 +5,21 @@ import com.anaplan.engineering.azuki.core.system.ActionFactory
 import com.anaplan.engineering.azuki.core.system.ParallelActionFactory
 import com.anaplan.engineering.azuki.core.system.UnsupportedAction
 
-interface GraphActionFactory<A: Action> : ActionFactory, ParallelActionFactory<A> {
+interface UndirectedGraphActionFactory<A : Action> : ActionFactory, ParallelActionFactory<A> {
```

Comment:
🚫 `UndirectedGraphActionFactory` is going to be one of the sub-objects.
```suggestion
interface GraphActionFactory<A : Action> : ActionFactory, ParallelActionFactory<A> {
```